html-tables/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>Student Grade Table</h1>
+    <table>
+      <tbody>
+        <tr>
+          <td><strong>Student Name</strong></td>
+          <td><strong>Course</strong></td>
+          <td><strong>Grade</strong></td>
+        </tr>
```

Comment:
This row should be in a `thead` and should be using `th` rather than `td`

Reply:
Hi @codingcodymiller I've modified the Student Grade Table with the thead element 
<img width="742" alt="Screen Shot 2021-12-01 at 10 03 22 AM" src="https://user-images.githubusercontent.com/76764048/144289037-71283e23-8470-400e-8c85-8df6150c3a75.png">
and used th rather than td!